src/spacewar/ui/view_frame.cljc

```diff
@@ -25,6 +25,13 @@
         [message-x (+ y h)]
         (q/text text 0 0)))))
 
+(defn fill-outside-rect [x y w h screen-w screen-h rgb]
+  (apply q/fill rgb)
+  (q/no-stroke)
+  (q/rect 0 0 x screen-h)
+  (q/rect x 0 w y)
+  (q/rect (+ x w) 0 (- screen-w w x) screen-h)
+  (q/rect x (+ y h) w (- screen-h h y)))
```

Comment:
The `fill-outside-rect` function and its use below is a kludge to make up for lack of clipping support in the ClojureScript version of quil. In short, it is used to achieve "poor-man's" clipping.